spec/lib/miq_expression_spec.rb

```diff
@@ -2423,6 +2423,21 @@
       expect(exp.to_human).to eq("COUNT OF Snaps > 1")
     end
 
+    it "generates a human readable string for a numeric FIELD value expression" do
+      exp = MiqExpression.new(">" => {"field" => "Vm.hardware.volumes-free_space_percent", "value" => 75})
+      expect(exp.to_human).to eq("VM and Instance.Hardware.Volumes : Free Space Percent > 75")
+    end
+
+    it "generates a human readable string for a float FIELD value expression" do
```

Comment:
in b4 "sort these tests"... it's sorted by expected formats (basic numeric first, then, float, then, symbol "to i with method" format).

🤣